hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -678,28 +702,30 @@
       name: chip_alert_handler_entropy
       desc: '''Verify the alert handler entropy input to ensure pseudo-random ping timer.
 
-
-            Details TBD.
+            - Force `alert_handler_ping_timer` input signal `wait_cyc_mask_i` to `4'bff` to shorten
+              the simulation time.
+            - Verify that the alert_handler can request EDN to provide entropy.
+            - Ensure that the alert ping handshake to all alert sources and escalation receivers
+              complete with errors.
```

Comment:
is this meant to mean "...complete with**out** errors" ?

Reply:
Ha! good catch, fixed.